validator/src/main.rs

```diff
@@ -1277,8 +1279,20 @@ pub fn main() {
                     exit(1);
                 }
             }
-        })
-        .collect();
+        }).map(
+        |account_path| {
+            // For all account_paths, set up the run/ and snapshot/ sub directories.
```

Comment:
Would be nice to update this comment to make it clear this will clean the accounts dir if they don't already exist.

Reply:
Added the comment "// If the sub directories do not exist, the account_path will be cleaned because older version put account files there".